app/jobs/split_pdf_job.rb

```diff
@@ -85,50 +102,51 @@ def save_pages(exam_template, partial_exams)
 
       group.access_repo do |repo|
         assignment_folder = exam_template.assignment.repository_folder
-        txn = repo.get_transaction(Admin.first.user_name)
-
+        unless Admin.first.nil?
```

Comment:
The only change I made from line 105 - 148 is putting existing codes inside unless Admin.first.nil? in order to handle error that occurs when Admin.first is nil.